android/app/src/main/java/updated/mysterium/vpn/ui/base/BaseActivity.kt

```diff
@@ -41,6 +43,7 @@ abstract class BaseActivity : AppCompatActivity() {
     protected var connectionState = ConnectionState.NOTCONNECTED
     protected val pushyNotifications = Notifications(this)
     private val dialogs = emptyList<Dialog>().toMutableList()
+    private val homeSelectionViewModel: HomeSelectionViewModel by inject()
```

Comment:
It's minor point. but we can make groupping of variables for clean. I mean group ViewModels separate, dialog separate some boolean state variable separate etc.